backend/de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/model/validator/ConcatenatePDFsCommand.java

```diff
@@ -103,8 +108,8 @@ private void enqueuePrintQueues(@NonNull final PrintingQueueQueryRequest queryRe
 
 		final I_C_Async_Batch asyncBatch = asyncBatchBL.newAsyncBatch()
 				.setContext(ctx)
-				.setC_Async_Batch_Type(C_Async_Batch_InternalName_AutomaticallyInvoicePdfPrinting)
-				.setName(C_Async_Batch_InternalName_AutomaticallyInvoicePdfPrinting)
+				.setC_Async_Batch_Type(queryRequest.getAsyncBatchType())
+				.setName(queryRequest.getAsyncBatchType())
```

Comment:
have a user-friendly name using `AD_Message` (and the currentl user's language)

Reply:
@metas-ts  (cc @LarsSander ) What should the name message contain?

Reply:
@metas-rc @LarsSander i would say "Dunning Run" / "Mahnlauf"